core/src/main/scala/org/bitcoins/core/wallet/utxo/InputInfo.scala

```diff
@@ -419,7 +447,9 @@ case class UnassignedSegwitNativeInputInfo(
     scriptWitness: ScriptWitness,
     conditionalPath: ConditionalPath,
     pubKeys: Vector[ECPublicKey])
-    extends InputInfo
+    extends InputInfo {
+  override def requiredSigs: Int = pubKeys.length
```

Comment:
I don't know the correct solution to this but I think this could cause problems

Reply:
Currently used only for fee estimation so better over than under, in the future this could change but generally speaking we only kind of support UnassignedSegwit anyway...